pkg/pipeline/encode/opentelemetry/encode_otlpmetrics.go

```diff
@@ -168,7 +168,7 @@ func NewEncodeOtlpMetrics(opMetrics *operational.Metrics, params config.StagePar
 				return nil, err
 			}
 			metricCommon.AddHist(histo, mInfo)
-		case "default":
```

Comment:
example of lack of type safety with the current model: "default"  has never been an allowed enum here. Now this would be caught at compile time.